setup.py

```diff
@@ -133,9 +139,24 @@ def tail_is(*suffixes):
 
 
 if __name__ == "__main__":
-    if not os.getenv("SKIP_CYTHON_BUILD"):
+    if not (os.getenv("SKIP_CYTHON_BUILD") or DEBUGPY_BUNDLING_DISABLED):
         cython_build()
 
+    # Etch bundling status in the source.
```

Comment:
I don't think this is necessary - the code can check at runtime whether `debugpy._vendored` is present by trying to import it and catching `ImportError`. This can be cached as a package global to make one-liner checks easier.

Also, if caching, I'd prefer the variable name to not use the `__reserved__` naming pattern - it can be simply `is_pydevd_bundled` or something like that. We declare our public API surface via `__all__`, so the name not starting with `_` is not a concern.